spec/formulaic/label_spec.rb

```diff
@@ -9,12 +9,70 @@
     expect(label(:user, :first_name)).to eq "First name"
   end
 
-  it "uses a translation if available" do
-    I18n.backend.store_translations(:en, { simple_form: { labels: { user: { name: "Translated" } } } } )
+  context "when simple_form translations are available" do
+    it "uses simple_form translation" do
+      I18n.reload!
```

Comment:
This setup is a bit repetitive; you might like to borrow Uniformity's implementation:

https://github.com/thoughtbot/uniformity/blob/master/spec/uniformity/integration_spec.rb#L242-L244
https://github.com/thoughtbot/uniformity/blob/master/spec/uniformity/integration_spec.rb#L242-L244